impl/core/src/main/java/io/serverlessworkflow/impl/executors/RunShellExecutor.java

```diff
@@ -128,21 +124,16 @@ public void init(RunShell taskConfiguration, WorkflowDefinition definition) {
               builder.environment().put(entry.getKey(), value);
             }
           }
-
           return builder;
         };
 
     this.shellResultSupplier =
         (taskContext, input, processBuilder) -> {
           try {
             Process process = processBuilder.start();
-
-            if (taskConfiguration.isAwait()) {
-              return buildResultFromProcess(taskConfiguration, definition, process);
-            } else {
-              return input;
-            }
-
+            return taskConfiguration.isAwait()
```

Comment:
yes, this one was not needed, but I love ternary operator and could not refrain, sorry ;)

Reply:
😆